src/components/wizard/Step_1/components/CreatePortfolio.vue

```diff
@@ -0,0 +1,9 @@
+<!--<template></template>-->
+
+<script>
+export default {
+  name: "CreatePortfolio.vue",
+};
+</script>
+
```

Comment:
Replace each of the script tags to properly implement typescript class based structure

`<script lang="ts">
import Vue from "vue";
import { Component } from "vue-property-decorator";

@Component({})
export default class Create Portfolio extends Vue {}
</script>`